modules/wp-cli/library.php

```diff
@@ -66,36 +66,76 @@ public function sync( $args, $assoc_args ) {
 	/**
 	 * Import template files to the Library.
 	 *
+	 *  [--returnType]
+	 *      Forms of output. Possible values are 'ids', 'info'.
+	 *      if this parameter won't be specified, the import info will be output.
+	 *
+	 *  [--fileProtocol]
+	 *      Forms of file source. Possible values are 'path', 'url'.
+	 *      if this parameter won't be specified, path will be the default.
```

Comment:
Why not check if it starts with `http`?

Reply:
We want to keep the old logic, and separate it from the new one (the url).